RELEASE.md

```diff
@@ -10,17 +10,29 @@ A new Grafana Mimir release is cut approximately every 6 weeks. The following ta
 | ------- | ---------- | ----------------- |
 | 2.0.0   | 2022-03-20 | Marco Pracucci    |
 | 2.1.0   | 2022-05-16 | Johanna Ratliff   |
-| 2.2.0   | 2022-06-27 | _To be announced_ |
+| 2.2.0   | 2022-06-27 | Oleg Zaytsev      |
 | 2.3.0   | 2022-08-08 | _To be announced_ |
+| 2.4.0   | 2022-09-19 | _To be announced_ |
+| 2.5.0   | 2022-10-31 | _To be announced_ |
 
 ## Release shepherd responsibilities
 
-The release shepherd is responsible for an entire minor release series, meaning all pre- and patch releases of a minor release. The process formally starts with the initial pre-release, but some preparations should be made a few days in advance.
-
-- We aim to keep the main branch in a working state at all times. In principle, it should be possible to cut a release from main at any time. In practice, things might not work out as nicely. A few days before the pre-release is scheduled, the shepherd should check the state of main. Following their best judgement, the shepherd should try to expedite bug fixes that are still in progress but should make it into the release. On the other hand, the shepherd may hold back merging last-minute invasive and risky changes that are better suited for the next minor release.
-- There may be some actions left to address when cutting this release. The release shepherd is responsible for going through TODOs in the repository and verifying that nothing is that is due this release is forgotten.
-- On the planned release date, the release shepherd cuts the first pre-release (using the suffix `-rc.0`) and creates a new branch called `release-<major>.<minor>` starting at the commit tagged for the pre-release. In general, a pre-release is considered a release candidate (that's what `rc` stands for) and should therefore not contain any known bugs that are planned to be fixed in the final release.
-- With the pre-release, the release shepherd is responsible for coordinating or running the release candidate in any end user production environment for at least 1 week. This is typically done at Grafana Labs.
+The release shepherd is responsible for an entire minor release series, meaning all pre- and patch releases of a minor release.
+The process formally starts with the initial pre-release, but some preparations should be made a few days in advance.
+
+- We aim to keep the `main` branch in a working state at all times. In principle, it should be possible to cut a release
```

Comment:
[not a blocker, but agreeing on some consistent will help long term maintaneability] I'm not sure this line break splitting follows docs squad recommendation. To my understanding there should be 1 full sentence per line, to help code reviews (so that with GitHub suggestions you can easily rewrite 1 sentence at a time). @jdbaldry am I right or wrong?

Reply:
I typicall stick to 1 sentence / line format, but got carried away in this list. Will fix it.

Reply:
I am definitely in favor of one line per sentence for easier line based diffing. I'm not yet certain that this is a strong enough argument to move from personal preference to enforced style though.